packages/strapi-plugin-content-type-builder/admin/src/containers/DataManagerProvider/tests/reducer_basic_actions.test.js

```diff
@@ -385,6 +385,14 @@ describe('CTB | containers | DataManagerProvider | reducer | basics actions ', (
         },
       },
     };
+    const singleTypes = {
```

Comment:
I don't think that adding the singleTypes to your test is relevant as you don't use the entry in your reudcer.